packages/components/src/components/search/_search.scss

```diff
@@ -240,6 +240,84 @@
       color: transparent;
     }
   }
+
+  .#{$prefix}--search--targeted__container {
+    display: flex;
+    align-items: stretch;
+
+    .#{$prefix}--dropdown__wrapper {
+      height: auto;
+    }
+
+    .#{$prefix}--dropdown {
+      flex: 0;
+      margin-right: rem(1px);
+    }
+
+    .#{$prefix}--search {
+      flex: 1;
+    }
+
+    .#{$prefix}--search--xl .#{$prefix}--search-magnifier {
+      left: auto;
+      right: 0;
+    }
+
+    .#{$prefix}--search--xl .#{$prefix}--search-input {
+      padding-left: rem(14px);
+      padding-right: rem(40px);
```

Comment:
just wondering, how are we deriving some of these values? typically I'm expecting spacing tokens or values equal to our spacing tokens but not sure if these are exceptions based on a spec. same with L300-301 and L313-314

Reply:
Good questions @emyarod, these are for adjusting the spacings to different sizes of search box. If designers have specific spacing tokens to use, we can make changes to use them. CC @designertyler @aagonzales

Reply:
I don't think either of those values are correct. 
padding-left: 16px / `$spacing-05`
padding-right: should be 48px so the icons is centered in an invisible square.

![image](https://user-images.githubusercontent.com/11670886/68810608-38d87f80-0634-11ea-927c-2276946c5db6.png)

Reply:
Thanks @aagonzales for clarifying - A follow-up question; What should happen with medium/small size variants? Or targeted search should have only the default size variant?

Reply:
it would stay centered in a squared container like this:

![image](https://user-images.githubusercontent.com/11670886/68889465-9a572780-06e2-11ea-8320-45e40b7addc9.png)